{{cookiecutter.repository_name}}/README.md

```diff
@@ -1,10 +1,116 @@
+<p align="left">
+<img width=15% src="https://dai.lids.mit.edu/wp-content/uploads/2018/06/Logo_DAI_highres.png" alt=“DAI-Lab” />
+<i>An open source project from Data to AI Lab at MIT.</i>
+</p>
+
 [![PyPI Shield](https://img.shields.io/pypi/v/{{ cookiecutter.package_name }}.svg)](https://pypi.python.org/pypi/{{ cookiecutter.package_name }})
 [![Travis CI Shield](https://travis-ci.org/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }}.svg?branch=master)](https://travis-ci.org/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }})
+{%- if cookiecutter.use_codecov == 'y' %}
+[![Coverage Status](https://codecov.io/gh/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }}/branch/master/graph/badge.svg)](https://codecov.io/gh/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }})
+{%- endif %}
+[![Downloads](https://pepy.tech/badge/{{ cookiecutter.package_name }})](https://pepy.tech/project/{{ cookiecutter.package_name }})
 
 # {{ cookiecutter.project_name }}
 
 {{ cookiecutter.project_short_description }}
 
+{% if cookiecutter.open_source_license == 'Not open source' -%}
 - Free software: {{ cookiecutter.open_source_license }}
+{% endif -%}
 - Documentation: https://{{ cookiecutter.github_owner }}.github.io/{{ cookiecutter.repository_name }}
 - Homepage: https://github.com/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }}
+
+# Overview
+
+TODO: Provide a short overview of the project here.
+
+# Install
+
+## Requirements
+
+**{{ cookiecutter.project_name }}** has been developed and tested on [Python {%- if cookiecutter.support_py2 == 'y' %}2.7, {%- endif %}3.4, 3.5, 3.6 and 3.7](https://www.python.org/downloads/)
+
+Also, although it is not strictly required, the usage of a [virtualenv](https://virtualenv.pypa.io/en/latest/)
+is highly recommended in order to avoid interfering with other software installed in the system
+where **{{ cookiecutter.project_name }}** is run.
+
+These are the minimum commands needed to create a virtualenv using python3.6 for **{{ cookiecutter.project_name }}**:
+
+```bash
+pip install virtualenv
+virtualenv -p $(which python3.6) {{ cookiecutter.repository_name }}-venv
+```
+
+Afterwards, you have to execute this command to have the virtualenv activated:
+
+```bash
+source {{ cookiecutter.repository_name }}-venv/bin/activate
+```
+
+Remember about executing it every time you start a new console to work on **{{ cookiecutter.project_name }}**!
```

Comment:
"Remember to execute it every time you start a new console!"

Reply:
I think that the original sentence is ok.

We do not need the user to activate the virtualenv every time a new console is started. We need that only when the console is started to work on the project :-)

Reply:
I had missed the first part of the sentence. I changed it accordingly, but kept the "to work on..." part.